workflow/rocoto/gfs_tasks.py

```diff
@@ -1186,25 +1186,38 @@ def _atmosoceaniceprod(self, component: str):
         return task
 
     def wavepostsbs(self):
+
         deps = []
         dep_dict = {'type': 'metatask', 'name': f'{self.run}_fcst'}
         deps.append(rocoto.add_dependency(dep_dict))
         dependencies = rocoto.create_dependency(dep=deps)
 
+        wave_post_envars = self.envars.copy()
+        postenvar_dict = {'FHR3': '#fhr#'}
+        for key, value in postenvar_dict.items():
+            wave_post_envars.append(rocoto.create_envar(name=key, value=str(value)))
+
         resources = self.get_resource('wavepostsbs')
-        task_name = f'{self.run}_wavepostsbs'
+        task_name = f'{self.run}_wavepostsbs_f#fhr#'
         task_dict = {'task_name': task_name,
                      'resources': resources,
                      'dependency': dependencies,
-                     'envars': self.envars,
+                     'envars': wave_post_envars,
                      'cycledef': self.run.replace('enkf', ''),
                      'command': f'{self.HOMEgfs}/jobs/rocoto/wavepostsbs.sh',
                      'job_name': f'{self.pslot}_{task_name}_@H',
                      'log': f'{self.rotdir}/logs/@Y@m@d@H/{task_name}.log',
                      'maxtries': '&MAXTRIES;'
                      }
 
-        task = rocoto.create_task(task_dict)
+        fhrs = self._get_forecast_hours('gefs', self._configs['wavepostsbs'])
+
+        member_var_dict = {'fhr': ' '.join([f"{fhr:03d}" for fhr in fhrs])}
```

Comment:
To avoid confusion, this should be fhr_metatask_dict
```suggestion
        fhr_metatask_dict = {'fhr': ' '.join([f"{fhr:03d}" for fhr in fhrs])}
```